app/upgrades/v4/constants.go

```diff
@@ -20,6 +24,9 @@ var Upgrade = upgrades.Upgrade{
 			consensustypes.ModuleName,
 			crisistypes.ModuleName,
 			lightclienttypes.ModuleName,
+			sponsorshiptypes.ModuleName,
+			dymnstypes.ModuleName,
+			irotypes.ModuleName,
```

Comment:
I believe we're missing `grouptypes.ModuleName`

Reply:
added